feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64ff0313700dad264d19dfe4.md

---
id: 64ff0313700dad264d19dfe4
title: Step 50
challengeType: 0
dashedName: step-50
---

# --description--

> Now that you have the `myTaskArr` array saved in `localStorage` in the right way, you can retrieve it with `getItem()` by passing in the key you saved the item with.

Reviewer: ```suggestion
Now that you have the `myTaskArr` array saved in `localStorage` correctly, you can retrieve it with `getItem()` by specifying the key you used to save the item.
```

> Use the `getItem()` method to retrieve the `myTaskArr` array and assign it to the variable `getTaskArr`. Log the `getTaskArr` variable to the console to see what happens.

Reviewer: ```suggestion
Use the `getItem()` method to retrieve the `myTaskArr` array and assign it to the variable `getTaskArr`. Then, log the `getTaskArr` variable to the console to see the result.
```
